src/argparse.lua

```diff
@@ -1290,27 +1290,36 @@ end
 
 function Parser:get_zsh_complete()
    local buf = {("compdef _%s %s\n"):format(self._name, self._name)}
-   table.insert(buf, ("_%s() {"):format(self._name))
-   table.insert(buf, "  typeset -A opt_args")
-   table.insert(buf, "  local context state line")
-   table.insert(buf, "  local ret=1\n")
-   table.insert(buf, "  _arguments -s -S -C \\")
+   table.insert(buf, "_" .. self._name .. "() {")
```

Comment:
Didn't like `:format`? (I'm fine either way, just curious why you changed)

Reply:
I was trying to avoid a function call inside `table.insert`